example/dune

```diff
@@ -0,0 +1,7 @@
+(executable
+ (name main)
+ (modes js)
+ (libraries incr_dom vg_canvas core_kernel)
+ (preprocess (pps js_of_ocaml-ppx ppx_jane))
+ (flags :standard -short-paths)
```

Comment:
The -short-paths declaration is no longer needed.

Reply:
Removed